server/schedule/filter/filters.go

```diff
@@ -152,14 +175,18 @@ func (f *excludedFilter) Type() string {
 	return "exclude-filter"
 }
 
-func (f *excludedFilter) Source(opt *config.PersistOptions, store *core.StoreInfo) bool {
-	_, ok := f.sources[store.GetID()]
-	return !ok
+func (f *excludedFilter) Source(opt *config.PersistOptions, store *core.StoreInfo) plan.Status {
+	if _, ok := f.sources[store.GetID()]; ok {
+		return regionExistedStatus
```

Comment:
regionExistedStatus may not be used directly, as evictLeader and labelScheduler use this filter to filter stores of unhealthy peers